packages/wallets/src/components/Base/WalletPasswordField/PasswordFieldUtils.ts

```diff
@@ -0,0 +1,30 @@
+export const handlePasswordScores = (passwordValue: string) => {
+    let score = 0;
+
+    const hasUpperCase = /[A-Z]/.test(passwordValue);
+    const hasLowerCase = /[a-z]/.test(passwordValue);
+    const hasDigit = /[0-9]/.test(passwordValue);
+    const hasSpecialChar = /[^A-Za-z0-9]/.test(passwordValue);
+    const hasMinLength = passwordValue.length >= 8 && passwordValue.length <= 25;
+
+    if (!hasMinLength) {
+        score++;
```

Comment:
Yea I understand it is a bit tricky to get it to work properly. Before this I was using the `zxcvbn` package however I am still not able to get the message to work properly. Maybe I should use a state to make it show for different scores.

Reply:
is the scoring used to decide if password is accepted, or is it intended to actually score password against security standard?

probably those two concerns - valdiation if password even can be accepted and scoring agains security standards, should be separated and done independanly. And scoring agains security standards probably only makes sense once the password fulfills validation to be accepted.

Reply:
would it make sense maybe to revisit that with designers and maybe use  the bar to indicate validation instead of score? reason being, that password scoring is suprisingly big topic

https://en.wikipedia.org/wiki/Password_strength

Reply:
Yes definitely! There is a better way of handling it but based on the design and requirements set by the qa I have to set the validation manually. Maybe next time we can try incorporating third libraries that handles these tasks once we have some feedback  from the designers on the issue.